src/config/config.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { plainToClass } from 'class-transformer';
+import { IsEnum, IsString, validateSync } from 'class-validator';
+
+@Injectable()
+export class ConfigService {}
```

Comment:
Ezt a config service-t igazából nem is fogjuk haszálni sehol, nem? mert a nestjs/config által adott service lesz behúzva ahol konfig import van, szóval ez itt törölhető.

Így sztem config module tartalma is változhatna, mivel nem lesz benne semmi nest specifikus dolog, ezért config module.ts sem kell, és én így csinálnék a config mappa alatt 2 fájlt:
validation.ts -> ez tartalmaza a validate függvényt, amit aztán az app module behúz.
environment-variables.ts -> ez pedig az environment enumot és a EnvironmentVariables osztályt, ami definiálja a formátumát a konfignak
ez a service fájl pedig így törölhető is